src/Backend/Modules/Extensions/Engine/Model.php

```diff
@@ -569,6 +569,26 @@ public static function getModulesThatRequireGoogleMaps()
     }
 
     /**
+     * Fetch the list of modules that require Google Recaptcha API key
+     *
+     * @return array
+     */
+    public static function getModulesThatRequireGoogleRecaptcha()
+    {
+        $modules = array();
+        $installedModules = BackendModel::getModules();
+
+        foreach ($installedModules as $module) {
```

Comment:
I could but I just applied the same style/logic as the methods before. Code consistency before all? And I don't feel like improving the other code cause that's one rabbit hole I would not like to dive in. (And you're going to change everything anyway)